README.md

```diff
@@ -105,6 +105,10 @@ Not set by default. If set to `true`, this plugin will download and process imag
 
 The plugin's default behavior is to fall back to Shopify's CDN.
 
+`typeName: string`
```

Comment:
My first question is whether or not this is the right name for this option. Is `typePrefix` better?

Reply:
Yeah, that makes more sense. I'll update!

Reply:
Changed!